app/src/main/java/com/example/tung/moviedb_07/data/source/remote/api/service/MovieApi.java

```diff
@@ -0,0 +1,42 @@
+package com.example.tung.moviedb_07.data.source.remote.api.service;
+
+import com.example.tung.moviedb_07.data.model.GenreList;
+import com.example.tung.moviedb_07.data.model.Movie;
+import com.example.tung.moviedb_07.data.model.MovieList;
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Created by tung on 6/13/17.
+ */
+
+public interface MovieApi {
+
+    String mApiKey = "api_key";
```

Comment:
ERROR: Static final field names must either be all caps (e.g. int HEIGHT_PX) for true constants, or start with s (e.g. AtomicInteger sNextId or Runnable sSuspendTask) for fields with mutable state or that dont feel like constants.